src/components/Company/CompanyTimeAndSalaryProvider.js

```diff
@@ -48,6 +62,14 @@ const CompanyTimeAndSalaryProvider = () => {
   const start = (page - 1) * PAGE_SIZE;
   const limit = PAGE_SIZE;
 
+  useEffect(() => {
+    dispatch(
+      queryCompanyTimeAndSalaryStatistics({
+        companyName: pageName,
+      }),
+    );
+  }, [dispatch, pageName]);
```

Comment:
does `queryCompanyTimeAndSalaryStatistics` need SSR?

Reply:
Thanks for the catch. Added